### ISSUE TITLE
api.dartlang.org doesn't work in Safari 8

### ISSUE THREAD

**Issue by sethladd** [dart-lang/dartdoc-viewer#31]
From https://github.com/dart-lang/dartlang.org/issues/980

**Comment by sethladd** [comment]
I went to go reproduce this, but I don't have Safari 8. :(

**Comment by mpgerlek** [comment]
Sorry, still failing with Safari 8.0.2 (verified does work with Chrome 40.0.2214.85), under OS X 10.10.1.

**Comment by sethladd** [comment]
We looked into this, here's what we found:

dartdocs.org generates docs for a package once. If new versions of polymer.dart are released, the fixes aren't retrofitted automatically to old docs.

New versions of dartdocs-viewer + polymer.dart are working in safari 8.

So, we've asked the maintainer of dartdocs.org to rebuild all docs with the latest polymer.dart + dartdocs-viewer. :)

I'll close this for now, as the issue is fixed with dartdoc-viewer. Many thanks for the report!
